src/NzbDrone.Core/Datastore/Migration/076_relax_not_null_constraints_extra_files.cs

```diff
@@ -0,0 +1,17 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(076)]
+    public class relax_not_null_constraints_extra_files : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Rename.Column("TrackFileId").OnTable("ExtraFiles").To("TrackFileIdLegacy");
+            Alter.Table("ExtraFiles").AddColumn("TrackFileId").AsInt64().Nullable();
+            Execute.Sql("UPDATE \"ExtraFiles\" SET \"TrackFileId\" = \"TrackFileIdLegacy\"");
+            Delete.Column("TrackFileIdLegacy").FromTable("ExtraFiles");
```

Comment:
Does AlterColumn not work correctly in this case?

```
Alter.Table("ExtraFiles").AlterColumn("TrackFileId").AsInt32().Nullable();
```

Reply:
Since SQLite doesn't support type changes I wrongly assumed that this needs to be manually handled in the migration. Apparently Fluent Migrator managed the change correctly. Fixed 👍